### PR TITLE
templates: Update CMS Guides for Education and Research

### PR DESCRIPTION
**Pull request by RaoOfPhysics**
If you want to see what these look like, I've taken screenshots of the full pages. Feedback welcome. Let me know if I should make any changes and push to this branch.

**Education**:

cms-guides-education

**Research**:

cms-guides-research

### CODE REVIEW

**Comment by RaoOfPhysics** [comment]
@tiborsimko, please let me know if hardcoding `<center>–––</center> is poor form. I can remove it if there's a better suggestion for how to break up the text within each "section".

**Comment by RaoOfPhysics** [comment]
Also: All the "quote" in the Research page are enclosed within `<blockquote>`s. So, we can give them a different style if we want.

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-education/education.md [hunk 1]
```diff
@@ -1,73 +1,67 @@
...
+    - [A quick (completely online) hands-on demo](https://mybinder.org/v2/gh/cms-opendata-education/cms-online-notebooks-for-binder/master?filepath=quick-start-to-CMS-open-data.ipynb) to Jupyter and CMS Open Data (using python).
+    - [Analysing CMS Open Data in Jupyter using **python**](TOBEFIXED)
+    - [Analysing CMS Open Data in Jupyter using **R**](TOBEFIXED), with a brief intro to the R programming language
 
-Have fun!!
+---
 
+#### Dive deeper into the data
 
+If you want a more detailed understanding of particle physics or an introduction aimed at the university level, take a look at some of the other resources in [CMS Learning resources ](http://opendata.cern.ch/collection/CMS-Learning-Resources).
```
Comment (sefeg) [inline cernopendata/modules/fixtures/data/articles/cms-guide-for-education/education.md]: @tiborsimko Are collection pages like _http://opendata.cern.ch/collection/CMS-Learning-Resources_ ported to _/search?q=CMS-Learning-Resources_ ?

**Comment by sefeg** [comment]
Looks very good to me. I leave a review decision on this to @katilp

**Comment by katilp** [comment]
Looks very good to me!

File: cernopendata/modules/fixtures/data/articles/cms-guide-for-education/education.md (hunk 1 above)
Comment (RaoOfPhysics) [reply cernopendata/modules/fixtures/data/articles/cms-guide-for-education/education.md]: @tiborsimko: Please let me know if I should change this before the PR is merged. Thanks!

**Comment by katilp** [comment]
Maybe replace the learning resource collection link with
http://opendata-qa.web.cern.ch/search?page=1&size=20&q=&keywords=education

**Comment by RaoOfPhysics** [comment]
Thanks for the feedback, @sefeg and @katilp. Now fixed in https://github.com/cernopendata/opendata.cern.ch/pull/1848/commits/701db9cb8a536398615676202b7d97c1424cb929

@ioannistsanaktsidis: I think this is good to go. I've rebased with the latest `master`.

**Pull request merged**